source/sdk/threading/native/ThreadWin32.ooc

```diff
@@ -68,13 +68,7 @@ version(windows) {
 	// http://support.microsoft.com/kb/104641/en-us
 	// CreateThread: extern func (...) -> Handle
 
-	version (gc) {
-		// Use Boehm-provided replacement for _beginthreadex
-		_beginthreadex: extern (GC_beginthreadex) func (security: Pointer, stackSize: UInt, startAddress: Pointer, arglist: Pointer, initflag: UInt, thrdaddr: UInt*) -> Handle
-	} else {
-		_beginthreadex: extern func (security: Pointer, stackSize: UInt, startAddress: Pointer, arglist: Pointer, initflag: UInt, thrdaddr: UInt*) -> Handle
-	}
-
+	_beginthreadex: extern func (security: Pointer, stackSize: UInt, startAddress: Pointer, arglist: Pointer, initflag: UInt, thrdaddr: UInt*) -> Handle
```

Comment:
I don't remember right now whether they have to be the same as in the library being called...